client/src/modules/stock/requisition/registry.js

```diff
@@ -244,6 +254,17 @@ function StockRequisitionController(
    * action to perform when the page is loaded
    */
   function startup() {
+
+    Status.read().then(rows => {
+      vm.status = rows.map(s => {
+        return {
+          value : s.status_key,
```

Comment:
This is awesome!

I propose two improvements:

1. Could you do the $translate.instant() here? It will help performance a bit.
2. Could you order to alphabetically after the $translate call? You could do it here with a sort() or in the template with an | orderBy: filter.

Reply:
I have troubles while trying to use $translate.instant() here, using an html template helped a lot.

Reply:
Alright, it isn't a bit enough deal to keep this out.  Thanks for trying!